tests/test_pyxis_task.py

```diff
@@ -0,0 +1,41 @@
+import json
+from unittest.mock import patch
+
+import pytest
+
+from corgi.core.models import Component, SoftwareBuild
+from corgi.tasks.pyxis import slow_fetch_pyxis_manifest
+
+pytestmark = pytest.mark.unit
+
+
+@pytest.mark.django_db
+@patch("corgi.tasks.sca.cpu_software_composition_analysis.delay")
+@patch("corgi.collectors.pyxis.session.post")
+def test_slow_fetch_pyxis_manifest(post, sca):
+    with open("tests/data/pyxis/manifest.json", "r") as data:
+        manifest = json.loads(data.read())
+    post.return_value.json.return_value = {"data": {"get_content_manifest": {"data": manifest}}}
+
+    image_id = "64dccc5b6d82013739c4f7b8"
+    slow_fetch_pyxis_manifest(image_id, SoftwareBuild.Type.BREW)
```

Comment:
The reason behind the second argument here isn't totally clear to me.

Reply:
That's an error! Good catch.

Reply:
Resolved in 442b443.